### PR TITLE
Renamed hook HOOK_ENTITY_CHANGE_WORLD

### PR DESCRIPTION
**Pull request by Seadragon91**
Please review and merge if the change is okay.

### CODE REVIEW

File: src/Entities/Entity.cpp [hunk 1]
```diff
@@ -1403,10 +1403,10 @@ bool cEntity::DoMoveToWorld(cWorld * a_World, bool a_ShouldSendRespawn)
 		return false;
 	}
 
-	//  Ask the plugins if the entity is allowed to change the world
-	if (cRoot::Get()->GetPluginManager()->CallHookEntityChangeWorld(*this, *a_World))
+	//  Ask the plugins if the entity is allowed to changing the world
```
Comment (SphinxC0re) [inline src/Entities/Entity.cpp]: This has to be 'change'

File: src/Entities/Entity.cpp [hunk 2]
```diff
@@ -1403,10 +1403,10 @@ bool cEntity::DoMoveToWorld(cWorld * a_World, bool a_ShouldSendRespawn)
...
+	if (cRoot::Get()->GetPluginManager()->CallHookEntityChangingWorld(*this, *a_World))
 	{
-		// A Plugin doesn't allow the entity to change the world
+		// A Plugin doesn't allow the entity to changing the world
```
Comment (SphinxC0re) [inline src/Entities/Entity.cpp]: This has to be 'change'

File: src/Entities/Player.cpp [hunk 3]
```diff
@@ -1606,9 +1606,10 @@ bool cPlayer::DoMoveToWorld(cWorld * a_World, bool a_ShouldSendRespawn)
 		return false;
 	}
 	
-	if (cRoot::Get()->GetPluginManager()->CallHookEntityChangeWorld(*this, *a_World))
+	//  Ask the plugins if the player is allowed to changing the world
+	if (cRoot::Get()->GetPluginManager()->CallHookEntityChangingWorld(*this, *a_World))
 	{
-		// A Plugin doesn't allow the player to change the world
+		// A Plugin doesn't allow the player to changing the world
```
Comment (SphinxC0re) [inline src/Entities/Player.cpp]: Same here

**Comment by madmaxoft** [comment]
okay, I missed a few typos in the comments. I wonder what important I missed :P

**Comment by SphinxC0re** [comment]
AFAIK nothing :+1:

**Pull request merged**